Templates/AppSource App/.github/workflows/CICD.yaml

```diff
@@ -144,11 +148,53 @@ jobs:
       buildMode: ${{ matrix.buildMode }}
       projectDependenciesJson: ${{ needs.Initialization.outputs.projectDependenciesJson }}
       secrets: 'licenseFileUrl,codeSignCertificateUrl,*codeSignCertificatePassword,keyVaultCertificateUrl,*keyVaultCertificatePassword,keyVaultClientId,gitHubPackagesContext,applicationInsightsConnectionString'
-      publishThisBuildArtifacts: ${{ needs.Initialization.outputs.workflowDepth > 1 }}
+      publishThisBuildArtifacts: true
```

Comment:
Why is this change needed?

_thisBuild_ artifacts are used for dependecy projects.

I suppose the ALDoc should use the 'official' published artifacts.

Reply:
wrong fix